Vagrantfile

```diff
@@ -79,5 +79,6 @@ Vagrant.configure(2) do |config|
        ansible_python_interpreter: "/usr/bin/python3",
     }
     ansible.raw_arguments = Shellwords.shellsplit(ENV['ANSIBLE_ARGS']) if ENV['ANSIBLE_ARGS']
+    ansible.raw_ssh_args = ['-o PubKeyAuthentication=yes -o DSAAuthentication=yes']
```

Comment:
out of interest, why do we need this. maybe add a comment above this to explain it

Reply:
I'll add a comment, but I explained it thoroughly in the commit message. Please do not squash these commits, if you don't mind.